bundles/core/org.eclipse.smarthome.core.test/pom.xml

```diff
@@ -23,22 +23,45 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>org.eclipse.tycho</groupId>
+        <groupId>${tycho-groupid}</groupId>
+        <artifactId>target-platform-configuration</artifactId>
+        <configuration>
+          <environments combine.self="override"></environments>
+          <dependency-resolution>
+            <extraRequirements>
+              <requirement>
+                <type>eclipse-feature</type>
+                <id>org.eclipse.smarthome.feature.test</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>org.eclipse.equinox.ds</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>org.eclipse.equinox.event</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>com.eclipsesource.jaxrs.jersey-min</id>
+                <versionRange>0.0.0</versionRange>
+              </requirement>
+              <requirement>
+                <type>eclipse-plugin</type>
+                <id>com.eclipsesource.jaxrs.jersey-all</id>
```

Comment:
> The feature description is "This features defines all bundles that needed to do OSGi based Unit testing.", so why not using it?

Because Jersey is only needed for REST. In the core bundle we don´t have any dependency to the REST, so it should definitely not be needed for executing the test. I just want to avoid unnecessary definition of dependencies. If this test bundle or other bundles really need it, something is wrong.